python/feathub/processors/processor.py

```diff
@@ -74,36 +125,14 @@ def get_table(
         pass
 
     @abstractmethod
-    def materialize_features(
-        self,
-        feature_descriptor: Union[str, TableDescriptor],
-        sink: FeatureTable,
-        ttl: Optional[timedelta] = None,
-        start_datetime: Optional[datetime] = None,
-        end_datetime: Optional[datetime] = None,
-        allow_overwrite: bool = False,
+    def execute_materialize_jobs(
+        self, materialize_job_descriptors: Sequence[MaterializeJobDescriptor]
     ) -> ProcessorJob:
         """
-        Starts a job to write a table of features into the given sink according to the
-        specified criteria.
+        Execute the given ProcessorJobGroup.
```

Comment:
Let's either change the input parameter to `ProcessorJobGroup`, or change this description to `execute the given jobs`/`execute the given job descriptors` so that this description matches with the signature.